src/fileAttachment.js

```diff
@@ -1,9 +1,17 @@
+import {require as requireDefault} from "d3-require";
+
 async function remote_fetch(file) {
   const response = await fetch(await file.url());
   if (!response.ok) throw new Error(`Unable to load file: ${file.name}`);
   return response;
 }
 
+async function dsv(file, format, {array = false, type = null} = {}) {
```

Comment:
Left a comment in the notebook about `type` being a bit too generic of an argument name here. I think I'd personally prefer `typed` but that's also not a home run either.

Reply:
I’ve updated it to `typed` and added `map` as a separate argument (that can be combined with `typed`, which is great because we couldn’t do that before!).